modules/extensions-spring-support/src/main/java/org/mule/module/extension/internal/config/BaseResolverFactoryBean.java

```diff
@@ -75,16 +74,4 @@ public void initialise() throws InitialisationException
     {
         LifecycleUtils.initialiseIfNeeded(valueResolver, muleContext);
     }
-
-    /**
-     * Propagates the disposal event to the
-     * underlying {@link #valueResolver} if it implements
-     * the {@link Disposable} interface
-     */
-    @Override
-    public void dispose()
```

Comment:
I don't understand. This code was there before your changes, what is the hack? And why did you remove the dispose and not the rest of the lifecycle methods?